patches/third_party-blink-public-mojom-webpreferences-web_preferences.mojom.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/third_party/blink/public/mojom/webpreferences/web_preferences.mojom b/third_party/blink/public/mojom/webpreferences/web_preferences.mojom
+index 973e7f926aac3c33545f723f0d4ade24759ae342..21a9452b8e9a51a1f8d9b9a998557e9a29228299 100644
+--- a/third_party/blink/public/mojom/webpreferences/web_preferences.mojom
++++ b/third_party/blink/public/mojom/webpreferences/web_preferences.mojom
+@@ -437,4 +437,6 @@ struct WebPreferences {
+ 
+   // Controls whether WebXR's immersive-ar is allowed.
+   bool webxr_immersive_ar_allowed;
++
++  bool allow_cosmetic_filtering;
```

Comment:
IMHO WebPreferences could be useful for use Brave features.
Maybe we could add struct BraveWebPreferences (with one bool) here?